src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -122,6 +124,19 @@ public void doPost_urlAlreadyInDatabase_shouldReturnLecture()
     verify(response).sendRedirect("/view/?id=1");
   }
 
+  @Test
+  public void doPost_videoHasNoTranscript_shouldReturnLectureAnyways()
+      throws IOException, ServletException {
```

Comment:
for tests do throws Exception.

Reply:
fixed!  I see a lot of other tests should make this change, so I created #355 to track